docs/governance.rst

```diff
@@ -0,0 +1,253 @@
+`decision-making rules for the BIDS standard <https://github.com/bids-standard/bids-specification/blob/master/DECISION-MAKING.md>`_, which in turn were inspired by the
+`lazy consensus system used in the Apache Foundation <https://www.apache.org/foundation/voting.html>`_,
+and heavily depend on the
+`GitHub Pull Request review system <https://help.github.com/articles/about-pull-requests/>`_.
+
+Definitions
+```````````
+
+Repository
+  `https://github.com/ME-ICA/tedana <https://github.com/ME-ICA/tedana>`_
+
+Contributor
+  Person listed in the `all-contributors file`_.
+  The community decides on the content of this file using the same process as any
+  other change to the Repository (see below) allowing the meaning of "Contributor"
+  to evolve independently of the Decision-making rules.
+
+Maintainer
+  A Contributor responsible for the long term health of the project and the
+  community. Maintainers have additional rights (see Rules) helping them to
+  resolve conflicts and increase the pace of the development when necessary.
+  Any maintainer can self-remove themselves. Any contributor can become a
+  maintainer by request and with the support of the majority of the current
+  maintainers. Current Maintainers:
+
+  +-----------------------------------+-----------------+
+  | Name                              | Time commitment |
+  +===================================+=================+
+  | `Logan Dowdle`_ (@dowdlelt)       | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Elizabeth DuPre`_ (@emdupre)     | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Dan Handwerker`_ (@handwerkerd)  | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Ross Markello`_ (@rmarkello)     | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Taylor Salo`_ (@tsalo)           | 3h/week         |
+  +-----------------------------------+-----------------+
+  | `Joshua Teves`_ (@jbteves)        | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Eneko Uruñuela`_ (@eurunuela)    | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Kirstie Whitaker`_ (@KirstieJane)| 0.5h/week       |
+  +-----------------------------------+-----------------+
+
+
+Rules
+`````
+
+1. Potential modifications to the Repository should first be proposed via an 
+   Issue. Rules regarding Votes apply to both Pull Requests and Issues.
+
+   - Every modification of the specification (including a correction of a typo, adding a new Contributor, an extension adding support for a new data type, or others) or proposal to release a new version needs to be done via a Pull Request (PR) to the Repository.
+2. Anyone can open a PR (this action is not limited to Contributors).
+3. PRs adding new Contributors must also add their GitHub names to the 
+   `all-contributors file`_. 
```

Comment:
I don't think we want anyone to edit that file directly !!!! Only the bot should do it.
I'd rather that we _offer_ (not force) folks to add their info to the Zenodo, and in that PR we can tag the bot to add them to the table.

Reply:
I think I might just remove this bullet point about PRs for new contributors. We're already saying elsewhere that contributors should be in the all-contributors file and that contrbutors are one thing that's added through PR. Why exactly do we need a specific rule that is seems overly detailed for this doc?